lib/semantic_puppet.rb

```diff
@@ -1,7 +1,18 @@
-require 'gettext-setup'
+begin
+  require 'gettext-setup'
+  GETTEXT_AVAILABLE = true
```

Comment:
The Puppet module has not been loaded here, what should we do about namespacing?